pom.xml

```diff
@@ -309,10 +309,10 @@
         <woodstox.stax2-api.version>4.2.1</woodstox.stax2-api.version>
         <wss4j.neethi.version>3.1.1</wss4j.neethi.version>
         <wss4j.version>2.3.4</wss4j.version>
-        <xalan.bundle.version>2.7.3_3</xalan.bundle.version>
-        <xalan.version>2.7.3</xalan.version>
-        <xerces.version>2.12.2</xerces.version>
-        <xerces.bundle.version>2.12.2_1</xerces.bundle.version>
+        <xalan.bundle.version>2.7.2_3</xalan.bundle.version>
```

Comment:
I just realized these were previously upgraded to resolve CVEs - downgraded here because the new CXF doesnt provide xalan and the servicemix.xalan-2.7.3 didnt it's dependencies like the 2.7.2 version. Will need to get this re-upgraded in a separate PR